backend/Testing/SyncReverseProxy/SendReceiveServiceTests.cs

```diff
@@ -173,10 +173,23 @@ await apiTester
         File.Exists(sendReceiveParams.FwDataFile).ShouldBeTrue();
 
         //hack around the fact that our send and receive won't create a repo from scratch.
-        HgRunner.Run("hg init", sendReceiveParams.DestDir, 1, new NullProgress());
-        HgRunner.Run("hg branch 7500002.7000072", sendReceiveParams.DestDir, 1, new NullProgress());
-        HgRunner.Run("hg add Lexicon.fwstub", sendReceiveParams.DestDir, 1, new NullProgress());
-        HgRunner.Run("""hg commit -m "first commit" """, sendReceiveParams.DestDir, 1, new NullProgress());
+        var progress = new NullProgress();
+        HgRunner.Run("hg init", sendReceiveParams.DestDir, 1, progress);
+        HgRunner.Run("hg branch 7500002.7000072", sendReceiveParams.DestDir, 1, progress);
+        HgRunner.Run($"hg add Lexicon.fwstub", sendReceiveParams.DestDir, 1, progress);
+        HgRunner.Run("""hg commit -m "first commit" """, sendReceiveParams.DestDir, 1, progress);
+
+        //add a bunch of small files, must be in separate commits otherwise hg runs out of memory. But we want the push to be large
+        const int totalSizeMb = 1100;
+        const int fileCount = 25;
+        for (int i = 1; i < fileCount; i++)
```

Comment:
I think you meant `<=` here:

```suggestion
        for (int i = 1; i <= fileCount; i++)
```